back/flag-service/src/flag/spec/FlagService.spec-integration.ts

```diff
@@ -65,6 +65,33 @@ describe('FlagService', () => {
     });
   });
 
+
+  describe('getPixel', () => {
```

Comment:
Description doesn't match `FlagService.getUserPixel()` method name. Also it'd be cool to have the same order of the methods in tests as in the class definition.

Reply:
I'd suggest to move up the `getUserPixel` method in `FlagService`'s definition to also match `PixelRepository`